.bumpversion.cfg

```diff
@@ -0,0 +1,16 @@
+[bumpversion]
+current_version = 0.4.0
```

Comment:
Note that the 0.4.0 tag doesn't exist currently, so not sure if this needs an adjustment or not.